paymentsheet/src/main/java/com/stripe/android/link/LinkLaunchMode.kt

```diff
@@ -18,5 +18,10 @@ internal sealed interface LinkLaunchMode : Parcelable {
      * to payment,
      */
     @Parcelize
-    data object Full : LinkLaunchMode
+    data class Full(
+        /**
+         * The selected Link payment method to be used for confirmation.
+         */
+        val selectedPayment: LinkPaymentMethod? = null
+    ) : LinkLaunchMode
```

Comment:
Any thoughts on introducing a different subclass instead of putting it into `Full`? Not a blocker, but we might want to change this as a fast-follow.